chromium_src/third_party/blink/renderer/core/dom/document.cc

```diff
@@ -6,6 +6,7 @@
 #include "third_party/blink/renderer/core/dom/document.h"
 
 #include "base/strings/string_number_conversions.h"
+#include "brave/renderer/brave_content_settings_agent_impl_farbling_helper.h"
```

Comment:
this is a layering violation, is there any reason why we can't just put `GetBraveFarblingLevel` in this file?